sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/networking.py

```diff
@@ -4,7 +4,7 @@
 
 from typing import Any, Dict, Optional
 
-from azure.ai.ml._restclient.v2023_02_01_preview.models import (
+from azure.ai.ml._restclient.v2022_12_01_preview.models import (
```

Comment:
@joshharrin As v2023-02-01_preview doesn't not have few of these APIs reverting to v2022-12.
Please help review and let me know if I miss any changes correspond to [this PR](https://github.com/Azure/azure-sdk-for-python/pull/28879)

Reply:
looks good now